framework/raw-aes-keyring.md

```diff
@@ -50,22 +50,10 @@ Galois/Counter Mode (GCM) Specification: [NIST Special Publication 800-38D](http
 
 On keyring initialization, the following inputs are REQUIRED:
 
-- [Key Namespace](#key-namespace)
-- [Key Name](#key-name)
+- [Key Namespace](./keyring-interface.md#key-namespace)
+- [Key Name](./keyring-interface.md#key-name)
 - [Wrapping Key](#wrapping-key)
 
-### Key Namespace
-
-A UTF-8 encoded value that, together with the [key name](#key-name), identifies a particular [wrapping key](#wrapping-key).
-
-This value is also used for bookkeeping purposes.
-
-### Key Name
-
-A UTF-8 encoded value that, together with the [key namespace](#key-namespace), identifies a particular [wrapping key](#wrapping-key).
-
-This value is also used for bookkeeping purposes.
-
```

Comment:
The equivalent lines should be removed from raw-rsa-keyring.

Reply:
I'm pretty sure those sections were not present in raw-rsa-keyring when I wrote this. It looks like they are in master now; addressed in merge from master.